sec.club/src/components/EventBanner/EventBanner.jsx

```diff
@@ -1,14 +1,22 @@
 import React from "react";
 import "./EventBanner.scss";
 
-export default function EventBanner({flyerSource, title, desc, endDate}) {
+export default function EventBanner({flyerSource, altText, title, desc, endDate}) {
     let flyer = require(`../../assets/flyers/${flyerSource}`)
+    let imageElement;
+
+    if (altText) {
+        imageElement = <img src={flyer.default} alt={`${altText}`}/>
+    } else {
+        imageElement = <img src={flyer.default} alt={`${flyerSource}`}/>
```

Comment:
IMO, the alt text should be more descriptive by default. Something like "Flyer for [Event]"